Bottom: ff8c8cd69d7c8182f3be6b4bd874f5c7c33c60de
Top:    3d9abdef891638d104879efc8edab2ed2c4d6b73
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-31 18:05:23 +0800

Refresh of debug.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 2008905..fc339de 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2142,6 +2142,8 @@ int main(int argc, char *argv[])
 				master = argv[2];
 		}
 
+		error(LOGOPT_ANY, "type %s name %s master %s", type, name, master);
+
 		if (master)
 			master_list = master_new(NULL, timeout, ghost);
 		else
@@ -2164,9 +2166,10 @@ int main(int argc, char *argv[])
 		master_list->nc = nc;
 
 		lookup_nss_read_master(master_list, 0);
-		if (type)
+		if (type) {
+			error(LOGOPT_ANY, "call dump_map");
 			dump_map(master_list, type, name);
-		else
+		} else
 			master_show_mounts(master_list);
 
 		head = &master_list->mounts;
